### PR TITLE
My auth

### PR DESCRIPTION
**Pull request by MarkyAaronYoung**

### CODE REVIEW

**Review by luketlancaster — COMMENTED** [review]
looks good to me, but there's conflicts. Needs master merged and then pushed up

**Review by psihanorak — APPROVED** [review]

File: src/javascripts/helpers/apiKeys.example.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "apiKey": "",
     "authDomain": "",
     "databaseURL": "",
-    "projectId": "",
+    "projectId": "",
```
Comment (luketlancaster) [inline src/javascripts/helpers/apiKeys.example.json]: extra space here should go away

File: src/javascripts/helpers/data/authData.js [hunk 2]
```diff
@@ -1 +1,21 @@
-import firebase from 'firebase/app'
+import firebase from 'firebase/app';
+import 'firebase/auth';
+
+const authDiv = $('#auth');
+const homeDiv = $('#homepage');
+const logoutButton = $('#navbar-logout-button');
+const checkLoginStatus = () => {
```
Comment (luketlancaster) [inline src/javascripts/helpers/data/authData.js:8]: give us a newline in between method definitions and variable definitions (just one right above this line is fine)

**Review by luketlancaster — CHANGES_REQUESTED** [review]
couple of nits that need to be picked. Should be easy enough

**Review by luketlancaster — APPROVED** [review]
lookin good!

**Review by psihanorak — APPROVED** [review]

**Pull request merged**